models/cobalt/model.sdf

```diff
@@ -168,6 +173,75 @@
       </velocity_decay>
     </link>
 
+    <link name='hydrophones'>
+      <pose> 0 0 0 0 0 0 </pose>
+      <inertial>
+        <mass> 0.001 </mass>
+      </inertial>
+      <visual name='ref_hydrophone_visual'>
+        <transparency>0.7</transparency>
+        <pose> 0.5 0 0 0 0 0</pose>
```

Comment:
They're arbitrary for now. Although I would recommend sliding the whole array back towards the rear of the sub so that they are outside of the camera's field of view.